en/gps_compass/u-blox_f9p_heading.md

```diff
@@ -29,14 +29,7 @@ Ideally the two antennas should be identical, on the same level/horizontal plane
 
 ### CAN Setup
 
-- The CAN port of the GPS devices need to be connected together and one connected to the autopilot.
-- Choose one GPS module to be the *Rover* and the other to be the *Moving Base*.
-  - On the *Rover*, set [GPS_UBX_MODE](../advanced_config/parameter_reference.md#GPS_UBX_MODE) to `1`.
-  - On the *Moving Base*, set [GPS_UBX_MODE](../advanced_config/parameter_reference.md#GPS_UBX_MODE) to `2`.
-- [EKF2_GPS_CTRL](../advanced_config/parameter_reference.md#EKF2_GPS_CTRL) parameter bit 7 must be set (see [GPS > Configuration > GPS as Yaw/Heading Source](../gps_compass/README.md#configuring-gps-as-yaw-heading-source)).
-- [GPS_YAW_OFFSET](../advanced_config/parameter_reference.md#GPS_YAW_OFFSET) may need to be set (see [GPS > Configuration > GPS as Yaw/Heading Source](../gps_compass/README.md#configuring-gps-as-yaw-heading-source)).
-- Reboot and wait until both devices have GPS reception.
-  `gps status` should then show the *Rover* GPS going into RTK mode, which means the heading angle is available.
+- See [ARK RTK GPS](../dronecan/ark_rtk_gps.md#setting-up-moving-baseline-gps-heading)
```

Comment:
@AlexKlimaj This is the top level document on CAN setup - the theory is that it is generic and other system link to it and share the information. Other vendors that support this feature are not going to want to on-link to ARK RTK documents.
It is OK to have duplicate material in ARK RTK GPS.

Can you revert this and make sure that it is up to date?

Reply:
At the moment, this setup is specific to the ARK RTK GPS. I think we just want this generic and to refer to the CAN RTK GPS docs to instructions.

Reply:
Thanks. I like your fix on this.